dstep/translator/CommentIndex.d

```diff
@@ -68,6 +68,73 @@ class CommentIndex
         {
             return offset < s ? -1 : (offset == s ? 0 : 1);
         }
+
+        private static bool isNormalized(string content)
+        {
+            import std.ascii : isWhite;
+
+            for (size_t i = 1; i < content.length; ++i)
+            {
+                if (content[i] == '\n' && content[i - 1].isWhite)
+                    return false;
+            }
```

Comment:
Not sure if it's better but you can:

1. Slice off the first character
2. Use `enumerate` to include the index of the current iteration
3. Use `canFind` with the same predicate as you have now